packages/cli/medusa-cli/src/create-cli.ts

```diff
@@ -170,13 +170,18 @@ function buildLocalCommands(cli, isLocalProject) {
       ),
     })
     .command({
-      command: `migrations [action]`,
+      command: `migrations [action] [modules...]`,
```

Comment:
suggestion: is it a common approach to have the spread on the right? otherwise I would suggest `[...modules]` wdyt?

Reply:
The yargs parser needs it at the end. So it is not only for the display purposes, it is to make their parse work